bigchaindb/util.py

```diff
@@ -501,6 +501,26 @@ def get_hash_data(transaction):
     return crypto.hash_data(serialize(tx))
 
 
+def verify_vote_signature(signed_vote):
+    """Verify the signature of a vote
+
+    A valid vote should have been signed `current_owner` corresponding private key.
+
+    Args:
+        signed_vote (dict): a vote with the `signature` included.
+
+    Returns:
+        bool: True if the signature is correct, False otherwise.
+    """
+
+    data = signed_vote.copy()
+
+    signature = data.pop('signature')
```

Comment:
I guess there is not need for the `copy` or the `pop`.
`public_key.verify(serialize(signed_vote['vote']), signed_vote['signature'])` should be enough.